test/cli_test.py

```diff
@@ -175,27 +185,30 @@ def test_clodius_aggregate_bedpe():
 
     assert(result.exit_code == 0)
 
-    tiles = cdt.get_2d_tiles(output_file, 0,0,0, numx=1,numy=1)
+    tiles = cdt.get_2d_tiles(output_file, 0, 0, 0, numx=1, numy=1)
     # print("tiles:", tiles)
 
-    assert('\n' not in tiles[(0,0)][0]['fields'][2])
+    assert('\n' not in tiles[(0, 0)][0]['fields'][2])
 
     #import json
```

Comment:
Just out of curiosity, why isn't this one changed?

Reply:
Good question, but I have no idea: all these white space changes were automatic.